xsnap/sources/xsnap-worker.c

```diff
@@ -544,6 +544,19 @@ int main(int argc, char* argv[])
 						fprintf(stderr, "%s\n", fxWriteNetStringError(writeError));
 						c_exit(E_IO_ERROR);
 					}
+					path = nsbuf + 1;
+					snapshot.stream = fopen(path, "rb");
+					if (snapshot.stream) {
+						fxUseSnapshot(machine, &snapshot);
+						fclose(snapshot.stream);
+					}
+					else
+						snapshot.error = errno;
+					if (snapshot.error) {
+						fprintf(stderr, "cannot restore snapshot %s: %s\n",
+								path, strerror(snapshot.error));
+						c_exit(E_IO_ERROR);
+					}
```

Comment:
As you know we went for another approach, so we'll be skipping this feature in that shape. I may consider enabling this as an explicit command however (e.g. `r`)

Reply:
Your call. My goal here is to make sure you have the option.